### ISSUE TITLE
 WSGI application 'project.wsgi.application' could not be loaded; Error importing module. 

### ISSUE THREAD

**Issue by mshemuni** [bugov/django-auto-logout#16]
When I tried to install and use the `django-auto-logout` I got an error.

# What I did

I followed the instructions [here](https://pypi.org/project/django-auto-logout/):

## Installation

`pip install django-auto-logout`

## Append to settings middleware:

```
MIDDLEWARE = [
...
    'django_auto_logout.middleware.auto_logout',
]
```

## REDIRECT_TO_LOGIN_IMMEDIATELY after the idle-time has expired

```
from datetime import timedelta
AUTO_LOGOUT = {
    'IDLE_TIME': timedelta(minutes=5),
    'REDIRECT_TO_LOGIN_IMMEDIATELY': True,
}
```

# The Error

```
Traceback (most recent call last):
  File "[SOME PATH]\site-packages\django\core\servers\basehttp.py", line 47, in get_internal_wsgi_application
    return import_string(app_path)
  File "[SOME PATH]\site-packages\django\utils\module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
  File "[SOME PATH]\site-packages\django\utils\module_loading.py", line 15, in cached_import
    module = import_module(module_path)
  File "[SOME PATH]\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\mshem\PycharmProjects\hilal\hilal\wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "[SOME PATH]\site-packages\django\core\wsgi.py", line 13, in get_wsgi_application
    return WSGIHandler()
  File "[SOME PATH]\site-packages\django\core\handlers\wsgi.py", line 125, in __init__
    self.load_middleware()
  File "[SOME PATH]\site-packages\django\core\handlers\base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
  File "[SOME PATH]\site-packages\django\utils\module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
  File "[SOME PATH]\site-packages\django\utils\module_loading.py", line 15, in cached_import
    module = import_module(module_path)
  File "[SOME PATH]\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "[SOME PATH]\site-packages\django_auto_logout\middleware.py", line 8, in <module>
    from .utils import now, seconds_until_idle_time_end, seconds_until_session_end
  File "[SOME PATH]\site-packages\django_auto_logout\utils.py", line 5, in <module>
    from pytz import timezone
ModuleNotFoundError: No module named 'pytz'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "[SOME PATH]\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "[SOME PATH]\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "[SOME PATH]\site-packages\django\utils\autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
    return get_internal_wsgi_application()
  File "[SOME PATH]\site-packages\django\core\servers\basehttp.py", line 49, in get_internal_wsgi_application
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: WSGI application '[My Project].wsgi.application' could not be loaded; Error importing module.
```

# Fix

Since it clearly says `ModuleNotFoundError: No module named 'pytz'` the fix was easy:

`pip install pytz`

**Comment by bugov** [comment]
Thank you. I will check and fix it this week.

**Comment by bugov** [comment]
This bug should be fixed in version 0.5.1 (will be available on pypi mirrors in a few minutes). Please check to be sure.
